src/styles/typography/index.md.njk

```diff
@@ -13,9 +13,14 @@ show_page_nav: true
 
 If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
 
+### When not to use the GDS Transport font
+
 You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.
```

Comment:
Can we match the "If" style of the first sentence, like this?
Also, just checking how 'publicly available' fits into this, does this look okay?

```suggestion
If your service is publicly available but on a different domain, use use an alternative typeface like Helvetica or Arial.
```

Reply:
Yeah, looks good! One tiny tweak:

```suggestion
If your service is publicly available but on a different domain, use an alternative typeface like Helvetica or Arial.
```

Reply:
Thanks, that looks good to me. I'll commit and do a sense check.

Reply:
@calvin-lau-sig7 Cheers! Any need to mention 'service.gov.uk,' like @hannalaakso suggested?